deploy/helm/sumologic/conf/metrics/collector/otelcol/config.yaml

```diff
@@ -67,6 +68,8 @@ receivers:
               - __address__
               target_label: instance
               action: replace
```

Comment:
maybe we can template it in helm :thinking:

Reply:
I don't think it's worth it for just two copies.